src/components/sessions/SessionTimeline.tsx

```diff
@@ -124,12 +127,25 @@ function SessionTimeline({
   handleTogglePublicEvaluation,
   handleResetQuestionBlock,
   handleActivateBlockById,
+  handleActiveBlock,
+  handleNoActiveBlock,
   subscribeToMore,
 }: Props): React.ReactElement {
   useEffect((): void => {
     subscribeToMore()
   }, [])
 
+  let isBlockActive = activeStep % 2 === 1
```

Comment:
Not sure why this is necessary, where do you use `isBlockActive` outside of the `useEffect`? It would be bad practice to alter a variable from within `useEffect`, so if that is needed here you would need to `useState`. But I think you could just remove this as `isBlockActive` is only used inside of `useEffect` and will always be computed on the first render as well.

Reply:
Alright, makes sense!